airflow/executors/kubernetes_executor.py

```diff
@@ -334,10 +334,15 @@ def process_error(self, event: Any) -> str:
             (raw_object['reason'], raw_object['code'], raw_object['message'])
         )
 
+
     def process_status(self, pod_id: str, status: str, labels: Dict[str, str], resource_version: str) -> None:
```

Comment:
`event` needs to be passed through here.

Reply:
@inytar thanks for this, I've pushed a commit to include it.